examples/F#/OpenGL/Tutorial 1.3 - Shaders/shader.fs

```diff
@@ -0,0 +1,48 @@
+module Shader
+
+open System
+open Silk.NET.OpenGL
+
+// Used to load and compile the vertex and fragment shaders
+let private createShader (gl: GL) (shaderType: GLEnum) path =

```

Comment:
> `Shader.use` doesn't really make sense either yet as you need to pass in everything to make that call and `use` is a reserved word so be `Shader.useProgram gl programId` just to call `gl.UseProgram(programId)`. Hopefully a more obvious abstraction will present itself in later tutorials

Note that in Fred's PR (#85), his shader abstraction caches the GL instance for later calls. I recommend trying to mimic Fred's abstraction as closely as possible. I've requested a review from him as well.

Reply:
Yeah, caching the gl instance seems a lot better and clearner to whoever would use the api, so i think it would be a good idea to teach good habbits, however; if it is more of a f# habbit/condition to pass it in every time, im happy with having it not be cached for the f# version

Reply:
In functional programming, pure functions don't rely on any external state. Pure functions also don't effect external state which is obviously not the case here as GL is one big state machine.
 However, I still think it's pretty idiomatic unless you start on the road of developing types to handle data that you would then eventually pass to OpenGL to process but that would create a lot of boilerplate and hide what we're really trying to teach.

Reply:
If we come up with a better pattern in later tutorials I'll revisit the earlier ones as well so we don't churn the code too much and keep the flow.